geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/membership/GMSJoinLeave.java

```diff
@@ -2706,6 +2715,7 @@ void prepareAndSendView(NetView newView, List<InternalDistributedMember> joinReq
       // can be transmitted to the new members w/o including it in the view message
 
       if (markViewCreatorForShutdown && getViewCreator() != null) {
+        markViewCreatorForShutdown = false;
```

Comment:
Since we have markViewCreatorForShutdown; how about adding unMark to set it false.